cedar-policy-validator/src/type_error.rs

```diff
@@ -241,7 +248,8 @@ pub enum TypeErrorKind {
     #[diagnostic(transparent)]
     UnexpectedType(UnexpectedType),
     /// The typechecker could not compute a least upper bound for `types`.
-    #[error("unable to find upper bound for types: [{}]", .0.types.iter().join(","))]
+    #[error(transparent)]
```

Comment:
I don't see a corresponding `#[error(...)]` below -- does miette use the `Display` trait automatically?

Reply:
I think it's the `Error` trait that uses `Display` ,and then `miette` uses `Error`, but yeah.